tests/test_eigen.py

```diff
@@ -744,6 +744,14 @@ def test_issue738():
     )
 
 
+def test_zero_length():
+    """Ignore strides on a length-0 dimension (even if they would be incompatible length > 1)"""
+    assert np.all(m.iss738_f1(np.zeros((0, 2))) == np.zeros((0, 2)))
```

Comment:
Strictly speaking, you should thank @EricCousineau-TRI for the tests 🙂; I just tweaked the ones he added in RobotLocomotion#38. (But, yeah, more tests == more good. Especially as Eric's original tests, which didn't cover all possible combinations of row-vs-column order and zero size, missed the numpy 1.23 breakage, which is why there are four rather than two. BTW, I also verified that this suite *does* catch the numpy 1.23 breakage, even with Eric's previous patch which only worked with numpy < 1.23.)

> this seems like it might be a good test case to parameterize using `@pytest.mark.parameterize`.

Do you mean something like this?
```python
@pytest.mark.parametrize("sizes", [(0, 2), (2, 0)])
def test_zero_length(sizes):
    """Ignore strides on a length-0 dimension (even if they would be incompatible length > 1)"""
    assert np.all(m.iss738_f1(np.zeros(sizes)) == np.zeros(sizes))
    assert np.all(m.iss738_f2(np.zeros(sizes)) == np.zeros(sizes))
```

Reply:
@mwoehlke-kitware Exactly!

Reply:
+1

The main advantage is that all tests run even if one fails. That's very useful especially when looking at CI logs, when it's not easy to rerun interactively to see what else fails.

Reply:
Example: test_const_name.py

Reply:
Agree with @rwgk, I was debating whether to parameterize on function names too, but this is pretty readable with that shorthand actually.

Reply:
Done.